assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -45,6 +50,10 @@ function DatePicker({ currentDate, onChange, onViewChange }) {
         value={value}
         onChange={handleOnChange}
         onViewChange={onViewChange}
+        nextAriaLabel={__('Next', 'web-stories')}
```

Comment:
Needs `_x()` context to make it clear what "Next" and "Previous" refers to. Year, month, week, day.

Depending on that the translations will be different.

Reply:
The label will be the same regardless of what it _next_'s to. So it should be a label, that'll work for both _next month_, _next year_, and _next decade_.

Otherwise we should request up-stream that this can be a function returning a label depending on the view, but that's a more complicated process.

Reply:
Yep, I'm not sure what could be a good label that would work for any. Maybe "Move to next" or "Move forward" or "Next dates" instead? These also don't sound quite right. Thoughts?

Reply:
Let's leave this worry as an exercise for the translator with an `_x()` context saying `"This label can apply to both next month, next year and/or next decade"` or similar.